src/api.js

```diff
@@ -0,0 +1,48 @@
+const listView = document.getElementById('list');
+
+/// Get data from API
+export const reload = async () => {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Zl4d7IVkemOTTVg2fUdz/scores/')
+    .then((response) => response.json())
+    .then((json) => {
+      listView.innerHTML = '';
+      for (let i = 0; i < json.result.length; i += 1) {
+        if (json.result[i].user === 'alan luqman') {
```

Comment:
- [ ] Kindly remove this filter on the name so any and all users can add their scores on your leaderboard and compare themselves to others. That's what makes this leaderboard such a fun project :)